product-scenarios/14-periodically-execute-an-integration-process/14.2-scheduling-a-customized-java-task/14.2.2-scheduling-a-java-task-with-cron/src/test/resources/keystores/key.crt

```diff
@@ -0,0 +1,15 @@
+-----BEGIN CERTIFICATE-----

```

Comment:
Shall we maintain keystores in scenario level?

Reply:
done